s2apler/timo/interface.py

```diff
@@ -141,7 +140,7 @@ def predict_one(self, instance: Instance) -> Prediction:
             mode="inference",
             n_jobs=self._config.n_jobs,
         )
-        if self._config.predict_incremental:
+        if instance.cluster_seeds:
```

Comment:
Why not use the config? It's more explicit...

Reply:
If we use the config, we'll need to have separate endpoint/deployment for incremental mode. It'll be nice for the clustering worker to send requests to the same endpoint. Also, this removes the possibility of user thinking they can send `cluster_seeds` in full block mode.

Reply:
OK, but can we add some kind of error message to catch this situation. How will a user know they can't send `cluster_seeds` in full block mode?

Reply:
The config is tied to the whole endpoint/deployment. It's not a parameter that can be specified on a per request basis.

Reply:
> OK, but can we add some kind of error message to catch this situation. How will a user know they can't send `cluster_seeds` in full block mode?

Yes, sure thing.

Reply:
I see. We probably just need to document this more extensively here in the comments then.